README.md

```diff
@@ -9,3 +9,24 @@ permalink: /
 
 ---------
 
+Facebook에서 공개한 React Native는 React의 방식을 이용하여 모바일 어플리케이션인 iOS와 Android를 동시에 만들 수 있는 멋진 프레임워크입니다.
+
+이 튜토리얼은 React Native 에서 꼭 필요한 `Basic`, `Debugging`, `React Navigation`, `State Management` 등을 다루고 있으며 각각의 챕터를 예제와 함께 진행하여 입문자들도 쉽게 따라할 수 있도록 만들어졌습니다. 
+
+‘React 를 알긴 알지만 잘 모르겠다’, 혹은 'React 를 아예 해본적 없다’라고 하셔도 상관없습니다. 
+기본적인 것을 다루고 있는 만큼, 해당 튜토리얼을 통해 React Native가 무엇인지, 어떻게 Javascript로 iOS와 Android 앱으로 개발할 수 있지? 등의 막연 했던 궁금증을 조금이라도 해결보세요.
+
+이 문서는 빠르게 변하는 React Native 프레임워크에 대응하기 위해 모두 오픈 소스로 공개되어 있습니다. 버그, 오타, 변경 사항 등의 이슈를 발견하시면 언제든지 [Issue](https://github.com/JeffGuKang/react-native-tutorial/issues) 에 남겨주시고, [기여](./Contribution.html)해주세요. 
+
+---------
+
+React Native, unveiled on Facebook, is a great framework for creating mobile applications iOS and Android at the same time using React.
+
+This tutorial covers the essential concepts of React Native, 'Basic', 'Debugging', 'React Navigation' and 'State Management', and each chapter is organized with an example so that beginners can easily follow easily.
+
+You can say, "I know the real thing, but I'm not sure." Or, "I don't know the real thing at all."
+Through this tutorial that we focus on the basics of real native, solve some vague questions like "What is Real Native?" and "How can you develop it with iOS and Android apps with Javascript?"
```

Comment:
`If you are new to React Native, that's fine.
This is a beginner-friendly tutorial so you can understand how React Native works and how it can build iOS and Android apps using Javascript.`